src/Interfaces/AbstractHPolygon.jl

```diff
@@ -123,11 +123,19 @@ function vertices_list(P::AbstractHPolygon{N};
         return points
     end
     @inbounds for i in 1:n-1
-        points[i] = element(intersection(Line(P.constraints[i]),
-                                         Line(P.constraints[i+1])))
+        cap = intersection(Line(P.constraints[i]), Line(P.constraints[i+1]))
```

Comment:
i presume this is not the best implementation in terms of efficiency; i would not pass through `EmptySet` and check the det condition directly. we can of course revisit this later, it's just a comment. the new function is already a step forward.

Reply:
Yes, this PR only intends to fix a bug.